elephant/test/test_spike_train_correlation.py

```diff
@@ -623,20 +626,30 @@ def _run_sub_tests(self, st1, st2, lags_true):
         for window in ('valid', 'full'):
             with self.subTest(msg="window={}".format(window),
                               window=window):
-                st1_binned = conv.BinnedSpikeTrain(st1, binsize=1 * pq.s)
-                st2_binned = conv.BinnedSpikeTrain(st2, binsize=1 * pq.s)
-                cch, bins = sc.cross_correlation_histogram(
+                binsize = 1 * pq.s
+                st1_binned = conv.BinnedSpikeTrain(st1, binsize=binsize)
+                st2_binned = conv.BinnedSpikeTrain(st2, binsize=binsize)
+                t_start_shift = (st2.t_start - st1.t_start) / binsize
+                t_start_shift = int(t_start_shift.simplified.magnitude)
+                left, right = lags_true[window][(0, -1), ] - t_start_shift
```

Comment:
> The true lags already take into account the differing `t_start` values

Exactly. That's why here I subtract `t_start_shift`. This is just a test.

> The input should be lags in terms of the real experimental time, the user does not have to worry about this shift.

Then the implementation might look confusing:
```python
# window list mode
left_edge, right_edge = window - t_start_shift
lags = np.arange(window[0], window[1], dtype=np.int32)

# in full mode we don't subtract t_start_shift
left_edge = window_min
right_edge = window_max
lags = np.arange(left_edge + t_start_shift, right_edge + 1 + t_start_shift, dtype=np.int32)
```

Do you think we should stick to this?

Reply:
I think that invoking `cch, lags = cross_correlation_histogram(bst1, bst2, window=(left, right))` should yield `lags[0] == left` and `lags[-1] == right`. Otherwise the interface is not very intuitive in my opinion.

Reply:
I agree, this is what users would expect.